index.js

```diff
@@ -41,10 +41,14 @@ var github = new GitHubApi({
     }
 });
 
-github.authenticate({
-    type: "oauth",
-    token: process.env.GITHUB_TOKEN
-});
+if(process.env.GITHUB_TOKEN){
+  github.authenticate({
+      type: "oauth",
+      token: process.env.GITHUB_TOKEN
+  });
+}else {
+  console.log('No GITHUB_TOKEN specified, do so to increase rate limit');
+}
```

Comment:
Very minor thing, mind having consistent spacing around the if/else and 4 space tabs, e.g.

```javascript
if (process.env.GITHUB_TOKEN) {
    github.authenticate({
        type: "oauth",
        token: process.env.GITHUB_TOKEN
    });
} else {
    console.log('No GITHUB_TOKEN specified, do so to increase rate limit');
}
```